.github/docker/Dockerfile.distroless-base-debian

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/base-debian12:latest@sha256:1dfdb5ed7d9a66dcfc90135b25a46c25a85cf719b619b40c249a2445b9d055f5
```

Comment:
Distroless images don't use tags so hoping that dependabot will be able to update these revisions.